src/event-helper.js

```diff
@@ -72,6 +72,23 @@ export function listenOnce(element, eventType, listener, opt_capture) {
 
 
 /**
+ * Unlistens for the specified event on the element.
+ * @param {?EventTarget} element
+ * @param {string} eventType
+ * @param {?function(Event)} listener
+ * @param {boolean=} opt_capture
+ */
+export function unlisten(element, eventType, listener, opt_capture) {
```

Comment:
There shouldn't be an `unlisten`. The `listen` and `listenOnce` return `Unlisten` function that can be called to unlisten. E.g.

```
const unlisten = listen(target, 'event', function() {});
// Later.
unlisten();  
```